electroncash/exchange_rate.py

```diff
@@ -300,8 +300,8 @@ def safe_float(value):
                 return 0
         # average of open,high,low,close prices. interpret garbage data as zero
         return {
-            i['Date']: (safe_float(i.get('Open', 0)) + safe_float(i.get('High', 0))
-                        + safe_float(i.get('Low', 0)) + safe_float(i.get('Close', 0))) / 4
+            i['Date']: (safe_float(i['Open']) + safe_float(i['High'])
+                        + safe_float(i['Low']) + safe_float(i['Close'])) / 4
```

Comment:
Yeah I can see why you'd want an exception here rather than `0` data.  Maybe this way if the API breaks it will be noticed.

Reply:
safe_float still makes it return 0 if it's invalid, the code i removed here was doing it a 2nd time. just some leftover code from testing. Probably best to remove the whole safe_float thing then as well.

The problem I experienced was the API returning null for 1 particular days worth of data on the BCH/USD pair, which stopped all the other data from showing. It seems OK now, so may have been a freak occurrence

Reply:
Ah I didn't realize that's what safe_float did -- true. Thanks for clearing that up.